docs/contributing/contributing-code/contributing-code-prerequisites/README.md

```diff
@@ -1,63 +1,85 @@
-- [jq](https://jqlang.github.io/jq/download/)  
-- Make  
+The steps to execute are:
+
+1. Press this button 
   
-  **Linux**: Install the `build-essential` package:
-  ```bash
-  sudo apt-get install build-essential
-  ```
-  **Mac**:
-  Using Xcode:
-  ```bash  
-  xcode-select --install
-  ```
-  Using Homebrew:
-  ```bash  
-  brew install make
-  ```
+    [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=340522752&skip_quickstart=true&machine=basicLinux32gb&devcontainer_path=.devcontainer%2Fcontributor%2Fdevcontainer.json&geo=UsWest)
+1. There is no second step - you are ready to contribute 😎
 
+> **Note** - When using GitHub Codespaces cost can occur on your side after you have used up the monthly included storage and core hours for your account. You find more details [here](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces).
 
-#### Testing Required Tools
+### VS Code and Dev Container
 
-If you have not already done so, clone the repository and navigate there in your command shell.
+To use the dev container you must have the following tools installed and running locally:
 
-You can build the main outputs using `make`:
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Dev Container Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+- [Docker](https://docs.docker.com/engine/install/)
 
-```sh
-make build && make lint
-```
+> 📝 **Tip** - If you are new to *dev containers* you find more information about developing inside a container [here](https://code.visualstudio.com/docs/devcontainers/containers) and a comprehensive tutorial [here](https://code.visualstudio.com/docs/devcontainers/tutorial)
 
-Running these steps will run our build and lint steps and verify that the tools are installed correctly. If you get stuck or suspect something is not working in these instructions please [open an issue](https://github.com/radius-project/radius/issues/new/choose).
+To get the dev container up and running
+
+1. If not already done, clone your fork of the Project Radius repository to your local file system and open the folder with VS Code. The best way to do so is either to launch VS Code and open the folder via the *File* -> *Open Folder* menu or to start VS Code from the command shell with `code .`, which opens the current directory as a folder in VS Code.
+
+1. Click the "X"-like button in VS Code to open a remote window
+
+    ![Button for opening remote window command palette](img/vscode-devcontainer-open-remote-button.png)
+
+1. Select the option "Reopen in Container" in the command palette
+
+    ![Remote window command palette](img/vscode-cmd-palette-container.png)
+
+The dev container will be started automatically.
+
+    ![Dev container startup process](img/vscode-devcontainer-opening-process.png)
```

Comment:
Not sure why but this is not rendering correctly. Here's what I see:

<img width="1349" alt="image" src="https://github.com/radius-project/radius/assets/1430011/bab65f4b-f2c5-4fa7-af52-1e9d31f756cb">

Reply:
I had a wrong indentation in the markdown. This is fixed now